src/parlant/core/engines/alpha/engine.py

```diff
@@ -139,6 +139,48 @@ async def process(
             self._logger.critical(f"Critical processing error: {traceback.format_exception(exc)}")
             raise
 
+    @override
+    async def utter(
+        self,
+        context: Context,
+        event_emitter: EventEmitter,
+        actions: Sequence[UtteranceRequest],
+    ) -> bool:
+        interaction_state = await self._load_interaction_state(context)
+        session_id = context.session_id
+
+        try:
+            with self._logger.operation(f"Uttering actions '{actions}' for session {session_id}"):
```

Comment:
Are we sure this log is correctly displaying the content of the utterance requests?